cli/Valet/Site.php

```diff
@@ -328,28 +328,43 @@ function pruneLinks()
     }
 
     /**
-     * Resecure all currently secured sites with a fresh tld.
+     * Resecure all currently secured sites with a fresh configuration.
      *
-     * @param  string  $oldTld
-     * @param  string  $tld
+     * @param  array  $old
+     * @param  array  $new
```

Comment:
I wonder if these arrays should have more documentation?
Something like:
There are only two supported values: `tld` and `loopback`
And those must be submitted in pairs else unexpected results may occur. eg: both $old and $new should contain the same indexes.